packages/client/src/pages/ProfilePage/ui/ProfilePage/ProfilePage.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ReactElement, useState, useEffect } from 'react';
+import { Button, Upload, Avatar, Row, Col, message } from 'antd';
+import { UploadOutlined } from '@ant-design/icons';
+import { useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { BaseLayout } from '@/layouts/BaseLayout';
+import { EAppRoutes } from '@/shared/types';
+import { useAppDispatch } from '@/shared/hooks';
+import { LogOut } from '@/features/LogOut';
+import { PasswordForm } from '@/widgets/PasswordForm';
+import { ProfileForm } from '@/widgets/ProfileForm';
+import { fetchUserInfo, uploadProfileAvatar, getUserData, getIsAuth } from '@/entities/User';
+import classes from './ProfilePage.module.scss';
+import { TEXTS, BASE_AVATAR_URL, DEFAULT_AVATAR } from './ProfilePage.constants';
+import { Navigate } from 'react-router-dom';
+
+export const ProfilePage = (): ReactElement => {
+  const isAuth = useSelector(getIsAuth);
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+  const user = useSelector(getUserData);
+  const [avatarLoading, setAvatarLoading] = useState(false);
+  const [isPasswordModalVisible, setIsPasswordModalVisible] = useState(false);
+
+  useEffect(() => {
+    dispatch(fetchUserInfo()).catch((error: Error) => {
+      console.error('Failed to fetch user info', error);
+      navigate(EAppRoutes.Auth);
+    });
+  }, [dispatch, navigate]);
+
+  const handleAvatarChange = async (file: File): Promise<void> => {
+    setAvatarLoading(true);
+    try {
+      // eslint-disable-next-line
+      await dispatch(uploadProfileAvatar(file) as any).unwrap();
```

Comment:
Я думаю, что unwrap надо обрать
```suggestion
      await dispatch(uploadProfileAvatar(file) as any).then((result) => {
      if (result.meta.requestStatus === 'fulfilled') {
       message.success(TEXTS.avatarUpdateSuccess);
      } else {
      message.error(TEXTS.avatarUpdateFailed);
      }
    });
```